src/main/kotlin/no/nav/sykdig/digitalisering/sykmelding/service/JournalpostService.kt

```diff
@@ -121,6 +123,32 @@ class JournalpostService(
         )
     }
 
+    fun ferdigstillAvvistOppgave(
+        oppgave: NasjonalManuellOppgaveDAO,
+        bruker: String,
+        navEnhet: String,
+        navEpost: String,
+        avvisningsgrunn: String?,
+    ) {
+        val oppgave = sykDigOppgaveService.getOppgave(oppgave.oppgaveId.toString())
+        val sykmeldt =
+            personService.hentPerson(
+                id = oppgave.fnr,
+                callId = oppgave.sykmeldingId.toString(),
+            )
+        val avvistGrunn = enumValues<Avvisingsgrunn>().find { it.name.equals(avvisningsgrunn, ignoreCase = true) }
+        if (avvistGrunn != null) {
+            sykDigOppgaveService.ferdigstillAvvistOppgave(
```

Comment:
Vi må kanskje ha ein eigen `ferdigstillAvvistOppgave` for nasjonal oppgave i dette tilfelle? `avvistGrunn` kan vere null og så trur eg ikkje vi kan gjenbruke `oppgaveRepository.ferdigstillAvvistOppgave`?

Reply:
Jepp, og oppgaveRepository skal ikke kalles